plugins/database/hana/hana_test.go

```diff
@@ -0,0 +1,172 @@
+package hana
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/vault/builtin/logical/database/dbplugin"
+	"github.com/hashicorp/vault/plugins/helper/database/connutil"
+)
+
+var (
+	testHANAImagePull sync.Once
+)
+
+func TestHANA_Initialize(t *testing.T) {
```

Comment:
I'm not aware of any officially supported docker images, although there may be a couple of examples in the wild. One thing to note is that HANA is entirely in-memory, and (I believe) requires 64GB or above ram to function.